_posts/2017-10-05-apprenticeship-at-8thlight.md

```diff
@@ -0,0 +1,213 @@
+
+I think that the student apprenticeship is a core thing. And is  one factor for the successful inclusivity as 8th Light practises it. The student apprenticeship allows and also encourages anyone (not only men) to try out and learn if becoming a developer can be a fit as their future profession. At some point in the discussions the [impostor syndrome] came up a couple of times. I believe that either of the apprenticeship programs, but especially the student apprenticeship do a great job of preventing the impostor syndrome to kick in. This means that the openness of the apprenticeship simply makes everybody welcome. No matter of their prior knowledge, experience and skill level. It lowers if not even removes the barrier to apply at a company that looks that inclusive. 
+
+[impostor syndrome]: https://en.wikipedia.org/wiki/Impostor_syndrome
+
+## The Apprenticeship
+
+The resident apprentice focuses on “programming, client services, collaboration, leadership, and communication skills” 
+as their [career site] states. Once graduated a crafter has obtained the skills to “learn to lead by example, provide and receive honest feedback, and answer the questions a client doesn't ask”.
+
+[career site]: https://8thlight.com/job-opportunities/london-crafter/
+
+The **apprenticeship is run like a real project**, like any other client project at 8th Light. That is the first learning: discipline. By running the apprenticeship like a project an apprentice lives and breathes all parts of a project iteration from the beginning and won’t be surprised when working with customers later.
+
+At the beginning the mentor creates a curriculum. (At some point I remember the ThoughtWorks learning path was mentioned too.) The curriculum might be a simple spread sheet that mentor and apprentice have access too. Mentor - that is the person that guides the apprentice through their apprenticeship, more about that later. The curriculum is the high-level plan for the apprenticeship and every week (that’s how long iteration take) more detailed tasks that are derived from the curriculum are put into a Kanban-like board. Theses tasks get estimated by the apprentice and depending on the acceptable load the next iteration’s “to-do” column is being filled. The board that we saw being used at 8th Light was a simple three column board with the columns backlog, to-do and done.
```

Comment:
The backlog is a separate (4th) column that is usually hidden. Three swimlanes are ready, working, and completed.